### ISSUE TITLE
Auto-upload is repeating file uploads if you do several uploads

### ISSUE THREAD

**Issue by JoaoSaleiro** [jkuri/ngx-uploader#316]
Hello,

1. Open http://ngx-uploader.com/ .
2. Choose auto-upload
3. Open the Network Monitor of your browser to track uploads
4. Choose one file
5. Confirm that network monitor shows 1 POST to the server (which is correct)
6. Choose another file
7. Check network monitor. The new file was POSTed twice to the server! (which is incorrect)
8. Choose another file
9. Check network monitor. The new file was POSTed three times to the server! (which is incorrect)

In other words, the auto-upload is not working correctly if you upload more than one file.

If you need further info, let me know. Also, let me know if you have a solution/workaround since this is affecting many users of this project :( 

Thank you!

**Comment by jkuri** [comment]
hi @JoaoSaleiro. thanks for this report! i will check on it during the weekend and find a solution hopefully.

**Comment by JoaoSaleiro** [comment]
Thank you jkuri! I hope you can fix it :) If you need help, or further info, please let me know!

**Comment by jkuri** [comment]
@JoaoSaleiro please check if it works for you and report in any case. thank you.

**Comment by GabrielMSharp** [comment]
@jkuri Working for me, thank you!

**Comment by JoaoSaleiro** [comment]
Hey @jkuri . I can confirm it's fine. Thanks! :)

**Comment by jkuri** [comment]
cool, thanks for report.
